spec/support/pageflow/dummy/rails_template.rb

```diff
@@ -53,7 +53,14 @@ def source_paths
 # ActiveAdmin does not look for admin definitions inside dummy apps by default.
 
 prepend_to_file('config/initializers/pageflow.rb', <<-END)
-  ActiveAdmin.application.load_paths.unshift(Dir[Rails.root.join('app/admin')].first)\n
+  ActiveAdmin.application.load_paths.unshift(Dir[Rails.root.join('admins')].first)\n
+END
+
+# add active admin patch to load pageflow activeadmin resources
+append_to_file('config/initializers/pageflow.rb', <<-END)
+  ActiveAdmin.after_load do |app|
+    Dir.glob("admins/**/*").each { |f| load(f) if File.file?(f) }
+  end\n
 END
```

Comment:
Why does [adding to `ActiveAdmin.application.load_paths`](https://github.com/codevise/pageflow/blob/0e9684524eaa46a757cbf35ec97efbdab14d7d35/lib/generators/pageflow/active_admin_initializer/active_admin_initializer_generator.rb#L10) no longer suffice?

Reply:
activeadmin specifies new way to add load_paths like this
`config.load_paths = [File.join(Rails.root, "app", "ui")]`
I tried it before with config.load_paths.unshift but somehow it didn't work, now I tried again for debugging and it works fine now, I am changing the source according to this config load_paths now

Reply:
for using unshift it works when i initialize load_paths with empty array like this
config.load_paths = []
If I dont initialize and print load_paths I get this
.rvm/gems/ruby-2.4.5/bundler/gems/cert_watch-a7cdd94ff724/admin
.rvm/gems/ruby-2.4.5/bundler/gems/pageflow-d2f784fe368b/admins
dedicated_pageflow/app/admin

If I initialize with empty array and then print, I get this
.rvm/gems/ruby-2.4.5/bundler/gems/cert_watch-a7cdd94ff724/admin
.rvm/gems/ruby-2.4.5/bundler/gems/pageflow-d2f784fe368b/admins

 it was not working because  dedicated_pageflow/app/admin does not exist

now that i initialize empty load_paths it works even with ActiveAdmin.application.load_paths
ActiveAdmin.application.load_paths = []

so now I can keep previous version and initialize load_paths with empty array in order to remove dedicated_pageflow/app/admin

Reply:
this works in hosted application but this is not working in pageflow dummy app so for now I am keeping it like this in pageflow and trying to understand what is causing it in pageflow dummy app

Reply:
To me it looks like [this commit](https://github.com/activeadmin/activeadmin/commit/2459474c6868aaab9b417fb14bd73345f6714f3d) added an initializer which runs after the `active_admin` initializer in the dummy app and resets the load path. If I change our code to 

```
Rails.application.config.after_initialize do
  ActiveAdmin.application.load_paths.unshift(Pageflow.active_admin_load_path)
end
```

the correct admin files are loaded again in pageflow specs and Dedicated. Not sure this is a stable solution, though. I think this is a bug in ActiveAdmin and we should probably file an issue.

I did not see a case yet where references to a non existing directory in the load paths caused an error. What kind of error message did you see?

Reply:
There was no error about non existing directory, I was just suspecting it because in our host application merge request it loads the files when I initialize load_paths to empty array.

Should I make changes according to this?
`Rails.application.config.after_initialize do
  ActiveAdmin.application.load_paths.unshift(Pageflow.active_admin_load_path)
end`

Reply:
Can you open an issue in the ActiveAdmin repository first and see if we get some feedback? I think this should be fixed in ActiveAdmin and we are not in a hurry to update at the moment. So if possible I would like to avoid working around the bug like that.

Reply:
sure